node_scenarios.py

```diff
@@ -0,0 +1,79 @@
+import sys
+import time
+import json
+import boto3
+import random
+import logging
+import kraken.kubernetes.client as kubecli
+import kraken.invoke.command as runcommand
+
+
+def node_start_stop_scenario(node_scenario_config):
```

Comment:
Maybe add a package for node scenarios and import it in the main file i.e run_kraken.py? Thoughts?

Reply:
Yeah, let's do that.